### ISSUE TITLE
DataHeader Write/Read-ability 

### ISSUE THREAD

**Issue by theNerd247** [theNerd247/JavaCSVReader#5]
Now that DataHeaders can be created and read from, lets clean up the format that we read data (no more accessing data through hard coding) and let's allow users to write and save data back into a CSV file.

**Comment by laserswald** [comment]
Perhaps we should make 2 classes for Reading and Writing? Then you can inherit from StreamReader and FileReader and all kinds of other stuff and get things for free! Inheritance rocks.

**Comment by theNerd247** [comment]
Nah, There is no point for an entire class. only to methods are needed and there is no inheritance going on. Also note that the API that we are working with is not the standard Java API. It's the only that FRC produces. I'll review the README file to make sure it explicitly states that.

**Comment by laserswald** [comment]
Oh yeah man i keep forgetting we are working with rocks and sticks here. 

Again, you would expect this low level stuff from C or C++, but Java? _What_?

Also, I requested a pull.

**Comment by theNerd247** [comment]
I saw the pull. You made changes that I already made in my repo. I'll be pushing to origin in a day or so. Also, the build file. I'm not using Ant to build my software. For the small stuff I'm doing it feels like overkill and I'm not going to be working with many Java programs for a while and see no need in learning it or using it. I have a custom form of building my files (a series of bash scripts - if want a copy I'll email them to you just lemme know). If want to include ANT build in this I'll put it on their, just know that I won't be managing it (I should probably create a disclaimer for it if that's the case).
